matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/Session.kt

```diff
@@ -245,6 +245,8 @@ interface Session :
 
     val sharedSecretStorageService: SharedSecretStorageService
 
+    fun getUIASsoFallbackUrl(authenticationSessionId: String): String
```

Comment:
Why is it directly available from the session, shouldn't be in some auth service ?

Reply:
It's OK for now